src/components/CurrentRefinements.vue

```diff
@@ -69,12 +69,16 @@ export default {
       return this.refinements.length === 0;
     },
     refinements() {
-      let refinements = this.state.refinements.map(item => ({
-        type: item.type,
-        attribute: item.type === 'query' ? 'query' : item.attributeName,
-        label: item.computedLabel,
-        value: item,
-      }));
+      let refinements = this.state.refinements.map(item => {
+        const attribute = item.type === 'query' ? 'query' : item.attributeName;
+        return {
+          type: item.type,
+          attribute,
+          label: item.computedLabel,
+          value: item,
+          key: `${attribute}-${item.computedLabel}`,
```

Comment:
computedLabel can change right? (in case of a query)
Isn't the key meant to be constant in vue?

Reply:
if it's a different query, it's a different element, we still need a unique key somehow, and the query will always have that issue

Reply:
Another option is to only add the suffix where duplicates are possible (disjunctive)

Reply:
FYI this is how I computed the key in InstantSearch.js 3:

https://github.com/algolia/instantsearch.js/blob/838abecceacad5bb72587697eb9c64642c325882/src/components/CurrentRefinements/CurrentRefinements.js#L5-L9

Reply:
Thanks @francoischalifour, we will revisit this when InstantSearch v3 will be incorporated here